src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -3,12 +3,14 @@
 import { Plate, TNode, usePlateEditorState } from "@udecode/plate";
 import React, { FC, useCallback, useEffect, useState } from "react";
 import { EditableProps } from "slate-react/dist/components/editable";
-import { debounce } from "../..";
+import { debounce, generateRandomId } from "../..";
 import { Toolbar } from "./Toolbar";
 import { getEditorConfig } from "./utils/getEditorConfig";
 import { TextStyleType } from "./utils/getTextStyles";
 import { EMPTY_VALUE, parseRawValue } from "./utils/parseRawValue";
 
+const defaultEditorId = generateRandomId();
```

Comment:
This is a sideeffect (not treeshakable), could you move this inside the component and use `useMemoizedId`?

Reply:
Changed to `useMemoizedId`